diff --git a/node_modules/azure-arm-network/lib/operations/routeTables.js b/node_modules/azure-arm-network/lib/operations/routeTables.js
index 5e3edbc..1bd8cd2 100644
--- a/node_modules/azure-arm-network/lib/operations/routeTables.js
+++ b/node_modules/azure-arm-network/lib/operations/routeTables.js
@@ -115,7 +115,7 @@ function _get(resourceGroupName, routeTableName, options, callback) {
     throw new Error('callback cannot be null.');
   }
   let expand = (options && options.expand !== undefined) ? options.expand : undefined;
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
@@ -444,7 +444,7 @@ function _list(resourceGroupName, options, callback) {
   if (!callback) {
     throw new Error('callback cannot be null.');
   }
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
@@ -579,7 +579,7 @@ function _listAll(options, callback) {
   if (!callback) {
     throw new Error('callback cannot be null.');
   }
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (this.client.subscriptionId === null || this.client.subscriptionId === undefined || typeof this.client.subscriptionId.valueOf() !== 'string') {
@@ -713,7 +713,7 @@ function _beginDeleteMethod(resourceGroupName, routeTableName, options, callback
   if (!callback) {
     throw new Error('callback cannot be null.');
   }
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
@@ -861,7 +861,7 @@ function _beginCreateOrUpdate(resourceGroupName, routeTableName, parameters, opt
   if (!callback) {
     throw new Error('callback cannot be null.');
   }
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
@@ -1042,7 +1042,7 @@ function _beginUpdateTags(resourceGroupName, routeTableName, parameters, options
   if (!callback) {
     throw new Error('callback cannot be null.');
   }
-  let apiVersion = '2019-02-01';
+  let apiVersion = '2019-09-01';
   // Validate
   try {
     if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
