diff --git a/node_modules/cruftless/lib/model/builder.js b/node_modules/cruftless/lib/model/builder.js
index 75d93a2..fe515fa 100644
--- a/node_modules/cruftless/lib/model/builder.js
+++ b/node_modules/cruftless/lib/model/builder.js
@@ -2,7 +2,7 @@
 (function() {
   var DOMParser, _, binding, curlyNS, forAllAttributes, xincludeNS, xsiNS;
 
-  DOMParser = require('xmldom').DOMParser;
+  DOMParser = require('@xmldom/xmldom').DOMParser;
 
   _ = require('lodash');
 
diff --git a/node_modules/cruftless/lib/model/element.js b/node_modules/cruftless/lib/model/element.js
index f0522f6..f006743 100644
--- a/node_modules/cruftless/lib/model/element.js
+++ b/node_modules/cruftless/lib/model/element.js
@@ -2,11 +2,11 @@
 (function() {
   var _, di, identity, parseExpr, parser, serializer, xsiNS;
 
-  serializer = new ((require('xmldom').XMLSerializer))();
+  serializer = new ((require('@xmldom/xmldom').XMLSerializer))();
 
-  di = new ((require('xmldom').DOMImplementation))();
+  di = new ((require('@xmldom/xmldom').DOMImplementation))();
 
-  parser = new ((require('xmldom').DOMParser))();
+  parser = new ((require('@xmldom/xmldom').DOMParser))();
 
   _ = require('lodash');
 
